src/zoom.js

```diff
@@ -9,16 +9,38 @@ export default (container, dimensions, scales, configuration, data, callback) =>
 
   const grid = d3.select('.grid');
 
-  if (configuration.slider) {
+  const sliderScale = d3.scale.log()
+    .domain([configuration.minScale, configuration.maxScale])
+    .range([configuration.minScale, configuration.maxScale])
+    .base(2);
 
+  const dropdown = d3.selectAll('.zoom-filter > li > a')
+    .on('click', zoomFilter);
+
+  // const datepicker = $('#datepicker')
```

Comment:
It would be a good practice to start deleting unused code instead of commenting it out.